core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -107,9 +107,11 @@ THE SOFTWARE.
 
         <!-- give actions a chance to contribute summary item -->
         <j:forEach var="a" items="${it.allActions}">
-          <st:include page="summary.jelly" from="${a}" optional="true" it="${a}" />
+          <div style="margin-top: 1em;">
```

Comment:
I'm guessing quite a lot more of breaking in plugins.
Ex: https://github.com/jenkinsci/build-failure-analyzer-plugin/blob/master/src/main/resources/com/sonyericsson/jenkins/plugins/bfa/model/FailureCauseBuildAction/summary.groovy

Reply:
Pretty sure I tested BFA and it rendered fine, can double check later

Reply:
Job page, browser allows the table and it looks the same as before:

![image](https://user-images.githubusercontent.com/21194782/111749055-3372f100-8889-11eb-841b-5a73eccfb725.png)

Run page, browser throws out the table tags, it looks fine except it's using a heading tag which is defined as a block element

![image](https://user-images.githubusercontent.com/21194782/111749132-471e5780-8889-11eb-9181-af8b6e410d7d.png)

We could maybe set an adjacent sibling selector to make headings display inline if after an `img` tag or svg, any thoughts @fqueiruga ?

Doesn't look like a blocker anyway to me, thoughts?

Reply:
Nah, that looks OK. Hopefully it won't break things rendered after it, though I don't think it will since it closes it's table tags.